src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sumOfAllNumbers = 0;
+  let copyOfSum;
```

Comment:
Variable `copyOfSum` is redundant. You can return the `sumOfAllNumbers` directly and then reset it to 0. This will make your code cleaner and more efficient.